internal/website/content/howto/docstore/_index.md

```diff
@@ -321,10 +321,24 @@ whose name is the combination of two or more fields.
 {{< goexample
 "gocloud.dev/docstore/mongodocstore.ExampleOpenCollectionWithIDFunc" >}}
 
+[`mongodocstore`]: https://godoc.org/gocloud.dev/docstore/mongodocstore
 [`mongodocstore.Dial`]: https://godoc.org/gocloud.dev/docstore/mongodocstore#Dial
 [`mongodocstore.OpenCollection`]: https://godoc.org/gocloud.dev/docstore/mongodocstore#OpenCollection
 [`mongodocstore.OpenCollectionWithIDFunc`]: https://godoc.org/gocloud.dev/docstore/mongodocstore#OpenCollectionWithIDFunc
 
+#### Azure Cosmos DB {#cosmosdb}
+
+[Azure Cosmos DB][] is compatible with MongoDB API. You can use the
+[`mongodocstore`][] package to connect to Cosmos DB. You must
```

Comment:
Can you separate this into a first section about how to use the URLOpener, and then a subheading for the constructor?

In particular, using the URLOpener, you would set the `MONGO_SERVER_URL` to the CosmosDB connection string (maybe link to https://docs.microsoft.com/en-us/azure/cosmos-db/connect-mongodb-account ?).

Reply:
I move the Azure section out and refer to both URL and constructors sections to MongoDB. Let me know whether this is what you meant.